src/Component/spec/Config/Builder/FieldSpec.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace spec\Sylius\Component\Grid\Config\Builder;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Grid\Config\Builder\Field;
+
+class FieldSpec extends ObjectBehavior
+{
+    function let(): void
+    {
+        $this->beConstructedWith('name', 'string');
+    }
+
+    function it_is_initializable(): void
+    {
+        $this->shouldHaveType(Field::class);
```

Comment:
Can we also check the interface existance?
```suggestion
        $this->shouldImplement(FieldInterface::class);
```